15_2_Mining_with_Regtest.md

```diff
@@ -0,0 +1,34 @@
+# 15.2: Mining the Regtest
+
+> **NOTE:** This is a draft in progress, so that I can get some feedback from early reviewers. It is not yet ready for learning.
+
+This document explains how to generate (i.e. mine) blocks using a Regtest (Regression Test) blockchain.
+To generate blocks on a new blockchain requires very minimal proof-of-work and it will take less than a second, due to the low difficulty and that it follows the Testnet rules.
```

Comment:
`take less than a second` is an understatement. It's basically instant. (You can generate 500 blocks in 1 second on a regular machine.)

`and that it follows the Testnet rules` is not really correct. It follows the regtest rules, which are separate from testnet.